scripts/consensus.py

```diff
@@ -33,21 +45,44 @@ def concatenate_data(profile_list: list) -> pd.DataFrame:
         concat_df.columns.str.startswith("Metadata")
     ]
     concat_metadata_df = concat_df.loc[:, concat_metadata_cols]
+
     concat_df = concat_df.drop(concat_metadata_cols, axis="columns")
-    concat_df = pd.concat([concat_metadata_df, concat_df])
+    concat_df = pd.concat([concat_metadata_df, concat_df], axis="columns")
 
     # dropping columns with na values
     na_cols = get_na_columns(concat_df, cutoff=0)
     concat_df = concat_df.drop(na_cols, axis="columns")
 
-    return concat_df
+    # aggregate
+    consensus_aggregate_config = consensus_config["aggregate"]
+    x_median_df = aggregate(
```

Comment:
ah! I see maybe your confusion. pycytominer has a consensus function:

https://github.com/cytomining/pycytominer/blob/master/pycytominer/consensus.py

Let's use this function instead of aggregate